### ISSUE TITLE
Apply visual customizations from a different NPC

### ISSUE THREAD

**Issue by llyarc** [focustense/easymod#24]
When you use many visual overhaul mods, there will be many eliminated faces from them. Although eliminated but their quality is still very high, is it possible to add features to record them and then use them on other NPC mod that do not have a good visual overhaul

**Comment by focustense** [comment]
If I understand you correctly, I think what you are asking about is the ability to take an overhaul option for one NPC, and apply that to a _different_ NPC? Sort of like a more efficient and GUI-friendly [NPC Visual Transfer Tool](https://www.nexusmods.com/skyrimspecialedition/mods/8764)?

**Comment by llyarc** [comment]
Yes, NPC Visual Transfer Tool is very troublesome when dealing with a large number of NPCs. So it would be very convenient if could be handled automatically according to the race, gender, etc.

**Comment by focustense** [comment]
Thanks for confirming.

Interesting request, and I think worth keeping on the roadmap, although I'm not sure when I might get to it. Copying or "moving" the face is actually the easy part; the hard part will be coming up with a UI that isn't confusing or tedious.

The basic idea of, say, "pinning" an NPC/face combo to some personal list, and then bringing that up in another screen, is somewhat obvious, but it's less obvious what to do when that list has hundreds of entries (or more), how to remove/categorize/search for entries, what to do when the plugins/mods are no longer present, or even what kind of UI to provide for actually pulling up NPCs that _don't_ have any direct customizations (right now they're filtered on startup).

This could be the type of thing that's better as a separate standalone app, or maybe a standalone sub-module or page within the app - one which doesn't interact much with the rest of the app, thus avoiding any conflicting assumptions. I'll give it some thought and leave any further ideas or mocks here.
